scripts/memory_logger.py

```diff
@@ -75,36 +57,36 @@ def _parseStackTrace(self, gibberish):
     else:
       return 'Stack Trace failed:', gibberish
 
-  def _waitForResponse(self, wait=True):
-    while wait:
-      self.gdb_stdout.seek(self.last_position)
-      for line in self.gdb_stdout:
+  def _waitForResponse(self, gdb_stdout, delta=float(5)):
```

Comment:
What's the 5 for? Is that seconds? Perhaps that should be a constant defined in this file somewhere?